lib/doh.c

```diff
@@ -884,7 +991,20 @@ doh2ai(const struct dohentry *de, const char *hostname, int port)
 #ifndef CURL_DISABLE_VERBOSE_STRINGS
 static const char *type2name(DNStype dnstype)
 {
+#ifndef USE_HTTPSRR
   return (dnstype == DNS_TYPE_A)?"A":"AAAA";
+#else
+  switch(dnstype) {
+    case DNS_TYPE_A:
+      return "A";
+    case DNS_TYPE_AAAA:
+      return "AAAA";
+    case DNS_TYPE_HTTPS:
+      return "HTTPS";
+    default:
+       return "unknown";
+  }
+#endif
```

Comment:
```suggestion
  switch(dnstype) {
    case DNS_TYPE_A:
      return "A";
    case DNS_TYPE_AAAA:
      return "AAAA";
#ifndef USE_HTTPSRR
    case DNS_TYPE_HTTPS:
      return "HTTPS";
#endif
    default:
       return "unknown";
  }
```

Reply:
done (well modulo ``s/ifndef/ifdef/`` in the middle of the switch)